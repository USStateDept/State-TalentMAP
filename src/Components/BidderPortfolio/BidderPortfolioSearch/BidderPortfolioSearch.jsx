import React from 'react';
import PropTypes from 'prop-types';
import ResultsSearchHeader from '../../ResultsSearchHeader';

const BidderPortfolioSearch = ({ onUpdate }) => (
  <div className="bidder-portfolio-search-container">
    <div className="results-search-bar-container">
      <ResultsSearchHeader
        labelSrOnly
        placeholder="Search Bidder Last Name, Skill cone (code)..."
        onUpdate={onUpdate}
      />
    </div>
  </div>
);

BidderPortfolioSearch.propTypes = {
  onUpdate: PropTypes.func.isRequired,
};

export default BidderPortfolioSearch;
