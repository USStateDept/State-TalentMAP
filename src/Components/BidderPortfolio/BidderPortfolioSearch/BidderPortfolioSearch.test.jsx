import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import BidderPortfolioSearch from './BidderPortfolioSearch';

describe('BidderPortfolioSearchComponent', () => {
  it('is defined', () => {
    const wrapper = shallow(
      <BidderPortfolioSearch
        onUpdate={() => {}}
      />,
    );
    expect(wrapper).toBeDefined();
  });

  it('matches snapshot', () => {
    const wrapper = shallow(
      <BidderPortfolioSearch
        onUpdate={() => {}}
      />,
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
