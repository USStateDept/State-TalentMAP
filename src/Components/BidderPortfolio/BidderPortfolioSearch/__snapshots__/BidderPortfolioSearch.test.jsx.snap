// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BidderPortfolioSearchComponent matches snapshot 1`] = `
<div
  className="bidder-portfolio-search-container"
>
  <div
    className="results-search-bar-container"
  >
    <ResultsSearchHeader
      defaultKeyword=""
      labelSrOnly={true}
      onFilterChange={[Function]}
      onUpdate={[Function]}
      placeholder="Search Bidder Last Name, Skill cone (code)..."
    />
  </div>
</div>
`;
